### PR TITLE
Add a few root files to lint-spell

### PR DESCRIPTION
**Pull request by silverwind**
Files in root were not linted, add them. No new violations.

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -146,7 +146,7 @@ WEB_DIRS := web_src/js web_src/css
 
 ESLINT_FILES := web_src/js tools *.config.js tests/e2e
```
Comment (silverwind) [inline Makefile]:
```suggestion
ESLINT_FILES := web_src/js tools *.js tests/e2e
```

**Comment by silverwind** [comment]
Should we spellcheck the changelog or do we consider it "immutable"? I'd say yes, we can change it.

**Comment by techknowlogick** [comment]
> I'd say yes, we can change it.

I agree

**Comment by silverwind** [comment]
Applied to changelog as well now and ran `make lint-spell-fix`. Also the `wildcard` is removed, which will give better Makefile performance.

**Comment by silverwind** [comment]
Ideally there would be a way to also lint commit messages during PR checks, not sure how to do it though, at least not without a temp file 😆.

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
> Ideally there would be a way to also lint the PR title during PR checks, not sure how to do it though, at least not without a temp file and then there's the issue that CI does not run after commit title change.

New @GiteaBot functionality? 🤔

**Comment by silverwind** [comment]
> > Ideally there would be a way to also lint the PR title during PR checks, not sure how to do it though, at least not without a temp file and then there's the issue that CI does not run after commit title change.
> 
> New @GiteaBot functionality? 🤔

Likely will be hard to run the misspell go spellchecker in the bot which is typescript. If there is a similar or stricter spellcheck tool that the bot could use, then yes, it would likely be rather easy to implement.

cc: @yardenshoham.

**Comment by yardenshoham** [comment]
hmmmm, maybe with https://www.npmjs.com/package/cspell

**Comment by silverwind** [comment]
I think I may actually exclude changelog again because I don't want to put the burden onto the releaser to fix typos that they haven't made.

**Comment by silverwind** [comment]
> hmmmm, maybe with https://www.npmjs.com/package/cspell

On topic: https://github.com/crate-ci/typos looks interesting, but it's rust which we have not integrated yet as tooling language.

**Comment by silverwind** [comment]
CHANGELOG.md is excluded again. I think we can at least keep the typo fixes in it, so it's a one-time correction basically. The `wildcard` does not seem to have any impact on Makefile performance, so I kept it.

**Review by lafriks — APPROVED** [review]

**Pull request merged**